proofOfConcept/ColorPickerAlpha/README.md

```diff
@@ -0,0 +1,9 @@
+<h1> Project Color Picker Alpha </h1>
+by Samantha Difeo, Tyrone Wong and Mathuran Sadagopan
+
+<h2>Overview</h2> 
+Have you ever struggled to sample the color of a picture or app on windows. Then project Color Picker Alpha is the solution for you! This app allows you to eyedrop the color of anything on your current window. By clicking anywhere on the screen the color at that point is immediately copied to you clip-board. When in RGB mode the color is displayed in RGB and will be copied to clip board in RGB.By selecting the toggle button the view and clip board switches to HEX.
```

Comment:
, then

Reply:
- clipboard, not clip-board
- Space before "By selecting"
- second time you wrote clip board, change to clipboard

Reply:
Have you ever struggled to sample the color of a picture or app on windows **? If so, then** project Color Picker Alpha is the solution for you! This app allows you to eyedrop the color of anything on your current window. By clicking anywhere on the screen the color at that point is immediately copied to you clip-board. When in RGB mode, **the color is displayed** and will be copied to clip board in RGB. By selecting the toggle button the view and clip board switches to HEX.